bedrock/firefox/templates/firefox/whatsnew/base-deprecated.html

```diff
@@ -0,0 +1,15 @@
+{#
```

Comment:
The idea here is to change the base extension in old WNPs to base-deprecated, and remove this file once all WNP before fx95 are deleted